presto-kafka/pom.xml

```diff
@@ -191,5 +206,12 @@
             <artifactId>kafka</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
https://github.com/prestosql/presto/pull/648#pullrequestreview-234652050

> Regarding usage of Mockito, we have a general policy of not using mocking libraries in Presto, as they tend to encourage bad practices when writing code, and they often result in brittle tests that depend on internal implementation details rather than external behavior.

Reply:
https://github.com/prestosql/presto/wiki/Code-style#mocking